frontends/aiger/aigerparse.cc

```diff
@@ -285,6 +285,8 @@ void AigerReader::parse_aiger()
 		}
 		else if (c == 'c') {
 			f.ignore(1);
+			if (f.peek() == '\r')
```

Comment:
Do you still need this if you're opening in `std::ifstream::binary`? If so, this would mean ABC is emitting '\r\n' right?